pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+latex: true
+---
+
+# OpenLoadFlow
+
+PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provided by PowSyBl. The source code is hosted on [GitHub](https://github.com/powsybl/powsybl-open-loadflow). 
+
+* TOC
+{:toc}
+
+## Network model in OpenLoadFlow
+
+The model used to perform sensitivity analysis is a classic model for load flow simulations.
+The network is represented by a graph whose nodes are the network buses and edges are the branch lines or transformers.
+The PI-model is used for branches and transformers.
+
+### PI-model for branches and transformers
+Test
+
+$$ a^2 = b^2 + c^2 $$
+
+### AC flows model
+
+### DC flows model
+
+## Sensitivity analysis for DC flows model
+
+
+## Configuration
+
+
+### Specific parameters
+
+**lowImpedanceBranchMode**  
```

Comment:
You cannot duplicate information here. Prefer a link to the power flow section.

Reply:
I think that you have already solved this point.